utils/loggingcpp/exceptclasses.h

```diff
@@ -263,6 +263,22 @@ class ProtocolError : public std::logic_error
 #ifndef __STRING
 #define __STRING(x) #x
 #endif
+#define idblog(x)                                                                       \
```

Comment:
idb means InfiniDB and that is the name of the project till 2014. Use mcs instead adding a new macro.

Reply:
Okay. I should remove it anyway.